documentation/user/Information center.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: wiki
+title: Information center
+description: Downloaded ACE3 and have no idea where to start? This page serves as a document to help new players get started with things or get an answer to some of your questions.
+group: user
+order: 0
+parent: wiki
+---
+
+Downloaded ACE3 and have no idea where to start? This page serves as a document to help new players and mission makers understand what's available to them.
+
+
+- You don't know where to begin your ACE3 journey? [**Check out ACE3 features**](http://ace3mod.com/wiki/feature/)
+
+
+- You are a mission maker but you don't know what ACE3 have to offer? [**We have some documentation for you**](http://ace3mod.com/wiki/missionmaker/)
+
+
+- Are you searching for ACE3 classnames ? [**Here they are**](http://ace3mod.com/wiki/missionmaker/classnames.html)
+
+
+## 1. FAQ
+### 1.1 Features 
+<b>Q:</b> Where is X feature? </br>
+<b>A:</b> When it's done.</br>
+
+<b>Q:</b> Feature X was in ACE2/AGM/CSE where is it?</br>
+<b>A:</b> It's going to be ported at some point.</br>
+
+<b>Q:</b> Why was my feature request was closed on GitHub?</br>
+<b>A:</b> Because it's supposed to go [HERE](https://github.com/acemod/ACE3/issues/414/)
+
+<b>Q:</b> I want to disable (feature) how do I do it?</br>
+<b>A:</b> Simply delete the PBO.(note that some features depends on others, check dependencies before deleting anything).</br>
+
+### 1.2 Issues
+
+<b>Q:</b> Laser locking is broken, when are you going to fix it?</br>
+<b>A:</b> Fun fact, it isn't, you need to come from the direction of the laser, (laser is pointing to the east, you come from the west) and you drop the GBU, it will then guide itself to the target.</br>
```

Comment:
Throw in something like: "because otherwise the laser would be obstructed by the vehicle and the aircraft would not be able to lock on"